src/SelectField/SelectField.jsx

```diff
@@ -197,6 +204,7 @@ const SelectField = React.createClass({
 
     return (
       <TextField
+        autoFocus={autoFocus}
```

Comment:
i jsut added it for DOM consistence...
if you add autoFocus to a Component imho the underlying form input should have autofocus @HTML lvl